hub/core/tests/test_meta.py

```diff
@@ -0,0 +1,143 @@
+from hub.htypes import DEFAULT_DTYPE, DEFAULT_HTYPE
+from hub.core.meta.tensor_meta import TensorMeta
+from hub.core.meta.index_meta import IndexMeta
+from hub.util.callbacks import CallbackList
+from hub.util.exceptions import (
+    MetaInvalidKey,
+    MetaInvalidRequiredMetaKey,
+    TensorMetaInvalidHtype,
+    TensorMetaInvalidHtypeOverwriteKey,
+)
+import pytest
+from hub.core.meta.dataset_meta import DatasetMeta
+from hub.core.meta.meta import Meta
+import hub
+
+
+TEST_META_KEY = "test_meta.json"
+
+
+def test_meta(local_storage):
```

Comment:
Can u add a test for "ds" as well and for other storage as well.

Reply:
sure